src/SecTester.Core/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -15,5 +19,28 @@ public static IServiceCollection AddSecTesterConfig(this IServiceCollection coll
       collection.Add(new ServiceDescriptor(typeof(Configuration), configuration));
       return collection;
     }
+
+    public static IServiceCollection AddSystemTimeProvider(this IServiceCollection collection, SystemTimeProvider? instance = null)
```

Comment:
Should we provide such atomic control of settings? I propose to register all of them with default options in `AddSecTesterConfig`, e.g.

```csharp
public static IServiceCollection AddSecTesterConfig(this IServiceCollection collection, string hostname)
{
  AddSecTesterConfig(collection, new Configuration(hostname));
  return collection;
}

public static IServiceCollection AddSecTesterConfig(this IServiceCollection collection, Configuration configuration)
{
  collection.Add(new ServiceDescriptor(typeof(Configuration), configuration));
  collection.AddSingleton(new UtcSystemTimeProvider());
  collection.AddLogging((builder) =>
  {
    // ...
    builder.SetMinimumLevel(configuration.LogLevel);
    // ...
  });
  return collection;
}
```